parlai/chat_service/services/terminal_chat/run.py

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env python3
```

Comment:
i'm unclear why this script exists and how it differs from the client? is it the server? Can you document it as such?